eng/pipelines/templates/stages/archetype-sdk-client.yml

```
@@ -73,6 +73,9 @@ parameters:
   type: boolean
   default: false
 
+variables:
+  - template: ../variables/globals.yml
```

```suggestion
  - template: /eng/pipelines/templates/variables/globals.yml
```
nit: Not necessary but makes copying around easier if this is relative to the root.